source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -3,13 +3,13 @@
 # This file is covered by the GNU General Public License.
 # See the file COPYING for more details.
 
+from __future__ import annotations
```

Comment:
could you comment what this is needed for?

Reply:
I've added a comment above the import. Is it OK like that?